src/tv2-common/cues/lyd.ts

```diff
@@ -45,8 +51,9 @@ export function EvaluateLYD(
 	}
 
 	const file = fade ? 'empty' : conf ? conf.FileName.toString() : parsedCue.variant
-	const fadeIn = fade ? Number(fade[1]) : conf ? Number(conf.FadeIn) : undefined
-	const fadeOut = conf ? Number(conf.FadeOut) : undefined
+	const fadeTimeInFrames = fade ? Number(fade[1]) : undefined
+	const fadeIn = fadeTimeInFrames ?? (conf ? Number(conf.FadeIn) : undefined)
+	const fadeOut = fadeTimeInFrames ?? (conf ? Number(conf.FadeOut) : undefined)
```

Comment:
I think `fadeOut` can stay unchanged.  the "empty" file does not need to fade out, only fade in

Reply:
I get your point.
But it will still get at fadeout in its transitions, it will just be whatever is configured as the fadeout time for the studio, which could be less than how long you want your fade to be (I have no idea though if this transition actually has an effect on the fading)
```
outTransition: {
  type: TSR.Transition.MIX,
  easing: TSR.Ease.LINEAR,
  direction: TSR.Direction.LEFT,
  duration: TimeFromFrames(fadeOut ?? config.studio.AudioBedSettings.fadeOut ?? 0)
}
```